integration/spark2/src/main/java/org/apache/carbondata/spark/vectorreader/ColumnarVectorWrapper.java

```diff
@@ -21,7 +21,7 @@
 import org.apache.spark.sql.execution.vectorized.ColumnVector;
 import org.apache.spark.sql.types.Decimal;
 
-public class ColumnarVectorWrapper implements CarbonColumnVector {
+class ColumnarVectorWrapper implements CarbonColumnVector {
```

Comment:
Because this is class is for vectorized reader feature in spark2 integration and It is only used in `VectorizedCarbonRecordReader` in the same package. It can be package-private instead of public